stanford_events_importer.install

```diff
@@ -760,3 +760,72 @@ function stanford_events_importer_update_7304(&$sandbox) {
     field_cache_clear();
   }
 }
+
+/**
+ * Update variables to point to events-legacy.stanford.edu.
+ */
+function stanford_events_importer_update_7305(&$sandbox) {
+  $category_source = variable_get('stanford_events_category_source');
+  if ($category_source == "http://events.stanford.edu/xml/drupal/v2.php?category-list") {
+    variable_set('stanford_events_category_source', 'https://events-legacy.stanford.edu/xml/drupal/v2.php?category-list');
+    watchdog('stanford_events_importer', 'Stanford Events Importer category list updated to pull from events-legacy.stanford.edu');
+  }
+  $organization_source = variable_get('stanford_events_organization_source');
+  if ($organization_source == 'http://events.stanford.edu/xml/drupal/v2.php?organization-list') {
+    variable_set('stanford_events_organization_source', 'https://events-legacy.stanford.edu/xml/drupal/v2.php?organization-list');
+    watchdog('stanford_events_importer', 'Stanford Events Importer organization list updated to pull from events-legacy.stanford.edu');
+  }
+}
+
+/**
+ * Update feeds_source config to point to events-legacy.stanford.edu.
+ */
+function stanford_events_importer_update_7306(&$sandbox) {
+  // See also stanford_events_importer_update_7201().
+  $configs = db_select('feeds_source', 'fs')
+    ->fields('fs',
+      array(
+        'feed_nid',
+        'config',
+        'source',
+      ))
+    ->condition('id', 'stanford_event_importer')
+    ->execute();
+
+  // Loop through each of the configured nodes and update the URL.
+  while ($values = $configs->fetchAssoc()) {
+    $nid = $values['feed_nid'];
+    $importer_config = unserialize($values['config']);
+    $source = $values['source'];
+    $source = str_replace('events.stanford.edu/xml/drupal/v2.php', 'events-legacy.stanford.edu/xml/drupal/v2.php', $source);
+    $importer_config['FeedsHTTPFetcher']['source'] = $source;
+
+    db_update('feeds_source')
+      ->fields(array(
+        'config' => serialize($importer_config),
+        'source' => $source,
+      ))
+      ->condition('feed_nid', $nid)
+      ->execute();
+  }
+  watchdog('stanford_events_importer', 'Stanford Events Importer feeds updated to pull from events-legacy.stanford.edu');
+
+}
+
+/**
+ * Update field_stanford_event_add_to_cal.
+ */
+function stanford_events_importer_update_7307(&$sandbox) {
+  $query = new \EntityFieldQuery();
+  $result = $query->entityCondition('entity_type', 'node')
+    ->entityCondition('bundle', 'stanford_event')
+    ->execute();
+  $nodes = node_load_multiple(array_keys($result['node']));
+  foreach ($nodes as $node) {
+    $add_to_cal_field = $node->field_stanford_event_add_to_cal[LANGUAGE_NONE][0]['url'];
```

Comment:
I just realized we should check for this field first before we do anything. The field might not exist or it might be empty